handlers/stock.js

```diff
@@ -0,0 +1,178 @@
+const azure = require('azure-storage');
+
+/**
+ * This handler manages the global supply of $chreef $tock.
+ * Each new user is given 1000 $$ once they join the channel.
+ * Use this stock as you will.
+
+ * Currently supports:
+ * Checking balance
+ */
+
+const TABLENAME = 'schreeftable';
+const ENTGEN = azure.TableUtilities.entityGenerator;
+const STARTINGSTOCK = 1000;
+
+module.exports = {
+    name: 'stock',
+    command: 'stock',
+    handler: stock
+};
+
+function stock(channel, message, params) {
+
+    // Wont work until this is populated from whatever is managing creds
+    var placeholderConnectionString;
```

Comment:
for local testing make a `.env` file and put env vars in it

for production, we need to add those vars to the heroku node

Reply:
I read some back and forth articles on it and bow to whatever style guide we choose. Converted fully to single at the moment and leaving it that way until for now.